src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -36,7 +36,7 @@
     private static final String REGEX_BLOB_KEY = "(encoded_gs_key:)?[a-zA-Z0-9-_]{10,}";
     private static final String REGEX_QUESTION_ID = "[a-zA-Z0-9-_]{40,}";
     private static final String REGEX_COMMENT_ID = "[0-9]{16}";
-    private static final String REGEX_DISPLAY_TIME = "(0[0-9]|1[0-2]):[0-5][0-9] [AP]M( UTC)?";
+    private static final String REGEX_DISPLAY_TIME = "(0[0-9]|1[0-2]):[0-5][0-9] [AP]M( -+[0-9])?";
```

Comment:
> Ideal case: detect the time zone and replace with ${datetime.now.utc+8}
As mentioned, I think currently we are doing the same only.With `new Date()` we are getting the date in system's time zone and replacing it with `${datetime.now}+0800`.